stock/templates/registered/srpfleetdata.html

```diff
@@ -27,92 +62,151 @@ <h1 class="page-header text-center">{% trans "SRP Fleet Data" %}
                     {% endif %}
                 </div>
             </h1>
-            <div class="alert alert-info" role="alert">
-                <div class="text-right">
-                    <b>{% trans "Total Losses:" %} {{ srpfleetrequests.count }}</b>
-                    &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
-                    <b>{% trans "Total ISK Cost:" %} {{ totalcost | intcomma }}</b>
-                </div>
-            </div>
+            
             {% if srpfleetrequests %}
-            <table class="table">
-                <tr>
-                    <th class="text-center">{% trans "Pilot Name" %}</th>
-                    <th class="text-center">{% trans "Killboard Link" %}</th>
-                    <th class="text-center">{% trans "Additional Info" %}</th>
-                    <th class="text-center">{% trans "Ship Type" %}</th>
-                    <th class="text-center">{% trans "Killboard Loss Amt" %}</th>
-                    <th class="text-center">{% trans "SRP ISK Cost" %}</th>
-                    <th class="text-center">{% trans "Post Time" %}</th>
-                    <th class="text-center">{% trans "Status" %}</th>
-                    {% if perms.auth.srp_management %}
-                        <th class="text-center">{% trans "Actions" %}</th>
-                    {% endif %}
-                </tr>
-                {% for srpfleetrequest in srpfleetrequests %}
+            <form method="POST">
+                {% csrf_token %}
+                <div class="alert alert-info" role="alert">
+                    <div class="text-right">
+                        <b>{% trans "Total Losses:" %} {{ srpfleetrequests.count }}</b>
+                        &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
```

Comment:
Maybe use `style="padding-right:3em"` instead. Then on the following conditional padding, use it on the button with `padding-left`. Alternatively wrap these in spans and apply it to that.